pb/payload.proto

```diff
@@ -28,44 +28,44 @@ enum PayloadType {
 }
 
 message Payload {
-	PayloadType     Type   = 1;
-	bytes           Data   = 2;
+	PayloadType     type   = 1;
+	bytes           data   = 2;
 }
 
 message Coinbase {
-	bytes Sender      = 1;
-	bytes Recipient   = 2;
-	int64 Amount      = 3;
+	bytes sender      = 1;
+	bytes recipient   = 2;
+	int64 amount      = 3;
 }
 
 message Commit {
-	bytes SigChain     = 1;
-	bytes Submitter    = 2;
+	bytes sigChain     = 1;
+	bytes submitter    = 2;
 }
 
 message DeleteName {
-	bytes  Registrant  = 1;
-    string Name        = 2;
+	bytes  registrant  = 1;
+    string name        = 2;
```

Comment:
There seems to be some inconsistency here (tab vs space)

Reply:
done.